src/coreclr/vm/bundle.cpp

```diff
@@ -64,6 +65,13 @@ BundleFileLocation Bundle::Probe(const SString& path, bool pathIsBundleRelative)
     pathBuffer.SetAndConvertToUTF8(path.GetUnicode());
     LPCSTR utf8Path(pathBuffer.GetUTF8());
 
+#if defined(TARGET_ANDROID)
+    // On Android we always strip away the assembly path, if any
```

Comment:
@grendello Thoughts on doing this stripping on the Android host side instead? For the runtime, I think it would make more sense to just pass the raw path to `external_assembly_probe`.

Reply:
Sure :+1: